### PR TITLE
Remove Intercom, Infantry Phone, and Vehicle Racks from parachutes

### PR DESCRIPTION
**Pull request by Drofseh**
**When merged this pull request will:**
- See title

### CODE REVIEW

**Comment by jonpas** [comment]
What do you have against parachutes?!

**Comment by Drofseh** [comment]
> What do you have against parachutes?!

They did a bad thing and lost their telephone privileges.

File: addons/sys_rack/CfgVehicles.hpp [hunk 1]
```diff
@@ -147,6 +147,10 @@ class CfgVehicles {
         };
     };
 
+    class ParachuteBase : Helicopter {
+        class AcreIntercoms {};
```
Comment (TheMagnetar) [inline addons/sys_rack/CfgVehicles.hpp]: Shouldn't it be  `class AcreRacks{};`

**Review by TheMagnetar — CHANGES_REQUESTED** [review]

File: addons/sys_rack/CfgVehicles.hpp (hunk 1 above)
Comment (Drofseh) [reply addons/sys_rack/CfgVehicles.hpp]: Derp, I copied and pasted for the format and forgot to change the class.

**Review by TheMagnetar — APPROVED** [review]

**Pull request merged**